packages/fx-core/src/common/telemetry.ts

```diff
@@ -71,6 +71,7 @@ export enum TelemetryEvent {
   CheckSubscription = "check-subscription",
   CheckSideloading = "check-sideloading",
   EnvConfig = "env-config",
+  OpenAdvancedProjectV3InV2 = "open-advanced-project-v3-in-v2",
```

Comment:
I'm not sure whether PM can easily understand what does this event mean. @KennethBWSong how's your feeling to this event name?

Reply:
not sure whether `display-tooling-update-notification` or something else is better?